tests/cases/compiler/abstractAsync.ts

```diff
@@ -0,0 +1,4 @@
+// @lib: es2015
```

Comment:
@rbuckton is `// @lib: es2015` the right thing or is there a leaner way to run this?